aibolit/model/model.py

```
@@ -173,17 +173,17 @@ def calculate_score(self, X, quantity_func='log', th=1.0, feature_importances=No
         except Exception:
             import traceback
             traceback.print_exc()
-            raise Exception("Unknown func")
+            raise Exception('Unknown func')
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)